documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,570 @@
+for accountability (to prevent projects from being stalled indefinitely) and for
+clarity of expectations for the other participants in the process. If the author
+requires additional time beyond their initial estimation, they should let the
+other participants know as soon as possible, keeping in mind that other
+discussions may be stalled in the meantime.
+
+To every extent possible, **no revisions should be made to the proposal before
+this point other than grammatical or spelling corrections**. The reason for this
+is to allow the question round to fully complete without the proposal changing
+during the discussion. This allows clarification to happen fully without having
+to track ongoing changes to the proposal and sets the expectation that changes
+the proposal incorporate all the clarifying questions asked in addition to any
+information surfaced in the reaction round. This helps to minimise the
+potentially "frantic" nature of a discussion that is happening about a proposal
+document that is changing simultaneously with the discussion.
+
+### Objection round
+
+This round lasts for 2 business days. Participants are now invited to share
+“paramount objections” to the proposal, if any exist. Objections are considered
+“paramount” only if:
+
+- They point to an issue that prevents the team from achieving its goals
+  (whether these are technical, community, or interpersonal). These issues can
+  also be said to "harm" the project or its contributors.
+- The objection reveals something that is irreversible (or extremely difficult
+  to reverse) of which the participants are not at least “okay” with or “sure
+  of”. For example, a proposal to switch to using MySQL when we’re currently
+  invested heavily in Postgres would require significant time investment that
+  would be difficult to reverse. If the participants are not absolutely certain
+  that it is the best decision, then any objection to making the switch would be
+  paramount.
+
+Note that during this round, like the question round, the author should not
+modify the text of the proposal. The focus of the author in this round should be
+in clarifying objections (especially paramount objections) and deciding whether
+it will be possible or reasonable to revise the proposal to address them. This
+allows time for participants to fully digest the proposal and raise all
+objections without further revision happening. Participants are expected to help
+each other decide whether an objection is paramount. Whether an objection is
+paramount can be questioned by the author, but they should prefer to defer to
+the participants and trust their judgement.
+
+If there are no paramount objections then the process has finished and the
+proposal is [ratified](#ratification). In that case mark the discussion as
+ratified, however this can be best done with the tool being used (merge the PR,
+close the discussion, etc). The author is responsible for updating the “Current
```

Comment:
From: 

> In that case mark the discussion as ratified, however this can be best done with the tool being used (merge the PR, close the discussion, etc).

To: 

> The proposal author will mark the discussion as ratified by merging their PR or closing their discussion.